<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DAK_Lab6.1</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_0">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Dossanov</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Adil</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Диванные войска</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Img">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Select img</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Img">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_0">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Фамилия</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сфера деятельности</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Run">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Навыки</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Мыслю позитивно</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ImgName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>185</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>не выбрано</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Опишите работу</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сколько лет в сфере</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Не создавать проблемы</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>21</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
